### PR TITLE
solution

### PR DESCRIPTION
**Pull request by maksym-kostetskyi**
https://maksym-kostetskyi.github.io/react_movies-list-add-form

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,16 +1,23 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
+import { Movie } from './types/Movie';
 import moviesFromServer from './api/movies.json';
 
 export const App = () => {
+  const [movies, setMovies] = useState<Movie[]>(moviesFromServer);
+  const addMovie = (newMovie: Movie) => {
```
Comment (GUSILLUS) [inline src/App.tsx:10]: Consider adding space here

File: src/components/NewMovie/NewMovie.tsx [hunk 2]
```diff
@@ -1,37 +1,128 @@
...
+
+  const [title, setTitle] = useState('');
+
+  const [description, setDescription] = useState('');
+
+  const [imgUrl, setImgUrl] = useState('');
+
+  const [imdbUrl, setImdbUrl] = useState('');
+
+  const [imdbId, setImdbId] = useState('');
```
Comment (GUSILLUS) [inline src/components/NewMovie/NewMovie.tsx:22]: You can group up states 
```suggestion
  const [count, setCount] = useState(0);
  const [title, setTitle] = useState('');
  const [description, setDescription] = useState('');
  const [imgUrl, setImgUrl] = useState('');
  const [imdbUrl, setImdbUrl] = useState('');
  const [imdbId, setImdbId] = useState('');
```

File: src/components/NewMovie/NewMovie.tsx [hunk 3]
```diff
@@ -1,37 +1,128 @@
...
+
+  const handleTitleChange = (event: string) => {
```
Comment (GUSILLUS) [inline src/components/NewMovie/NewMovie.tsx:24]: Let's rename the function attribute to `value` to clarify the logic

File: src/components/NewMovie/NewMovie.tsx [hunk 4]
```diff
@@ -1,37 +1,128 @@
...
+        required={true}
+      />
 
       <div className="field is-grouped">
         <div className="control">
           <button
             type="submit"
             data-cy="submit-button"
             className="button is-link"
+            disabled={!title || !imgUrl || !imdbUrl || !imdbId}
```
Comment (GUSILLUS) [inline src/components/NewMovie/NewMovie.tsx:125]: Create a variable for this conditional to improve readability

**Review by GUSILLUS — CHANGES_REQUESTED** [review]
GJ! Let's improve code style a little

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Consider disabling the submit button if there are just spaces 
Screenshot 2024-04-09 at 14 41 47

File: src/components/NewMovie/NewMovie.tsx [hunk 5]
```diff
@@ -1,37 +1,132 @@
...
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [imgUrl, setImgUrl] = useState('');
+  const [imdbUrl, setImdbUrl] = useState('');
+  const [imdbId, setImdbId] = useState('');
+  const checkDisable =
+    !title ||
+    !imgUrl ||
+    !imdbUrl ||
+    !imdbId ||
```
Comment (anastasiiavorobiova) [inline src/components/NewMovie/NewMovie.tsx:22]:
```suggestion

```

**Review by anastasiiavorobiova — APPROVED** [review]
Great work!

**Pull request closed without merging**